### ISSUE TITLE
Crash when installing mods w/ sound preview via 1-click install

### ISSUE THREAD

**Issue by Radfordhound** [thesupersonic16/HedgeModManager#52]
HMM crashes consistently with the following information when trying to install [this mod](https://gamebanana.com/sounds/44711) from GameBanana via 1-click install:

```
HMM Info:
    Version: 7.0-328
    Args: -gb hedgemmforces:https://gamebanana.com/mmdl/424822,Sound,44711
    StartDir: E:\Program Files (x86)\Steam\steamapps\common\SonicForces\build\main\projects\exec\
    Process Level: User
    Game: Sonic Forces
    SteamGame: Sonic Forces

Exception:
    Type: ArgumentNullException
    Message: Value cannot be null.
Parameter name: value
    Source: Newtonsoft.Json
    Function: System.Object DeserializeObject(System.String, System.Type, Newtonsoft.Json.JsonSerializerSettings)
    StackTrace: 
       at Newtonsoft.Json.JsonConvert.DeserializeObject(String value, Type type, JsonSerializerSettings settings)
       at Newtonsoft.Json.JsonConvert.DeserializeObject[T](String value, JsonSerializerSettings settings)
       at HedgeModManager.UI.GBModWindow.Window_Loaded(Object sender, RoutedEventArgs e)
       at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
       at System.Windows.UIElement.RaiseEventImpl(DependencyObject sender, RoutedEventArgs args)
       at System.Windows.BroadcastEventHelper.BroadcastEvent(DependencyObject root, RoutedEvent routedEvent)
       at System.Windows.BroadcastEventHelper.BroadcastLoadedEvent(Object root)
       at MS.Internal.LoadedOrUnloadedOperation.DoWork()
       at System.Windows.Media.MediaContext.FireLoadedPendingCallbacks()
       at System.Windows.Media.MediaContext.FireInvokeOnRenderCallbacks()
       at System.Windows.Media.MediaContext.RenderMessageHandlerCore(Object resizedCompositionTarget)
       at System.Windows.Media.MediaContext.RenderMessageHandler(Object resizedCompositionTarget)
       at System.Windows.Interop.HwndTarget.OnResize()
       at System.Windows.Interop.HwndTarget.HandleMessage(WindowMessage msg, IntPtr wparam, IntPtr lparam)
       at System.Windows.Interop.HwndSource.HwndTargetFilterMessage(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
       at MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
       at MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)
       at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
       at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
       at System.Windows.Threading.Dispatcher.LegacyInvokeImpl(DispatcherPriority priority, TimeSpan timeout, Delegate method, Object args, Int32 numArgs)
       at MS.Win32.HwndSubclass.SubclassWndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam)
    InnerException: 
```

As you can see I'm on the latest rewrite version.
Looks like it crashed because the given mod features a sound on its GameBanana page and HMM doesn't know how to deal with that.

**Comment by thesupersonic16** [comment]
Thanks for posting this issue. I have partially fixed the issue in 3a7562bb7b6b27fc43f053c3947dbb72b501109c, Right now it just excludes the `screenshot` field when loading sounds, but something that would be cool is some kind of media player to play those previews in the future.

**Comment by Sajidur78** [comment]
b1188ae65a8222de4db9bc5a1bb5c6ecc5df5994 Fixes this by implementing a sound player to HMM
